### PR TITLE
PanelChrome: New logic-less emotion based component with no dependency on PanelModel or DashboardModel

### PR DESCRIPTION
**Pull request by torkelo**
This early work trying to extract the styles and visualization aspect of the panel chrome to a new component with emotion styles that has no logic or dependency on PanelModel & DashboardModel.  (Plan is to rename PanelChrome and PanelChromeAnguar to something else). 

Plan is to then experiment with new layout and styles for panel header & icons & panel links. 

super early state 
Screenshot from 2020-12-04 14-59-09

### CODE REVIEW

**Comment by mckn** [comment]
panel_chrome

File: packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+export const PanelChrome: React.FC<PanelChromeProps> = ({
+  title = '',
+  children,
+  width,
+  height,
+  padding = 'md',
+  leftItems = [],
+}) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
```
Comment (torkelo) [inline packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx:26]:
```suggestion
  const styles = useStyles(getStyles);
```

**Review by torkelo — COMMENTED** [review]
Looks good!

**Review by mckn — APPROVED** [review]
Adding a review for the "Looks good" from @torkelo

**Review by peterholmberg — CHANGES_REQUESTED** [review]
Very nice @mckn ! It would be nice with a story with controls too so I as a user can change behavior without changing the code.

**Comment by torkelo** [comment]
I think all the behaviors are already exposed in the different examples

**Review by peterholmberg — APPROVED** [review]

**Pull request merged**